### ISSUE TITLE
[NEW BUILD - DEFAULT / 20w17a] Starship EVO land! Backers build!

### ISSUE THREAD

**Issue by tsunamayo** [tsunamayo/Skywanderers#1788]
This build is doing the switch to Starship EVO.
As the name of the app changed you need to copy your blueprint over to
C:\Users\Francois\AppData\LocalLow\Moonfire Entertainment\Starship EVO\Save_Data\Blueprints
you can also copy SETTINGS.ini from 
C:\Users\Francois\AppData\LocalLow\Moonfire Entertainment\Starship EVO

Beware this build breaks the instance format! 
So you will need to create a new character, and make sure all your save where exported to blueprint, you will need to re-import them.
Worst case if you didnt have the blueprint and already made the update: go to
C:\Users\Francois\AppData\LocalLow\TsunamayoCorp\Skywanderers\Save_Data\PlayerEntities
and find (by looking at the size and the last saved date) blueprint matching you ship. You can then copy the .mps over the new blueprint folder and import it.

Here is what you will find in the build:

- New Save logic: each character has its own world (separate entities)
- New logo and background screens
- New music, for now only in the title screen
- autosave (every 60sec), and save on exit
- Building drone is dead. Next update will bring the default color back.
- Improved Starter block: anti-grav, can load a blueprint. I will continue to improve it.
- New Simple Seat as this was requested a lot!
- New mini cushion tile that rescale, for your custom seat contruction ;)

I am working on the kickstarter post!
Cheers

**Comment by Sullos** [comment]
Hey tsuna, just wondering how we go about deleting a new build in this new version? 
I loaded a ship blueprint in, and the same goes for all blueprints, and it was the wrong one and I can't seem to find out how to delete a build.

**Comment by Sullos** [comment]
Also Tsuna, just letting you know, in case you are unaware, on steam within our Steam Library, the game name doesn't appear as Starship EVO, it still appears as Skywanderers.

![unknown](https://user-images.githubusercontent.com/33239800/79962607-9ae2be00-84cb-11ea-917d-f4ea577ece48.png)

**Comment by tsunamayo** [comment]
Yes thanks I know, I need to contact steam again I cant change it on my own despite what they first told me.
For the delete indeed there is no option atm as you cant put the starter block on an entity.
I will add that feature, so the way of doing it will be: put a starter block, then open the screen then delete it. Hope that is not too clunky...
In survival that will be possible at shipyard...
Thanks

**Comment by Sullos** [comment]
I think the idea for placing the starter block would work perfectly and is no problem at all. 
Maybe a new name would be needed in the future though as its more of a 'Building Core' now?
As with the way it could be used to build hovercrafts and other things as it was all discussed in the suggestion post the other day.
Thanks

**Comment by Battlepixel** [comment]
But then you can not save a blueprint in-game without saving the ship with the build block installed?
I think this should be possible.

**Comment by tsunamayo** [comment]
Hum good point @Battlepixel =( I will see what I can do...
-> either auto save to blueprint
-> or remove all build block.
Not very clean I agree...
Yes it should be renamed, building core is quite good!

**Comment by Battlepixel** [comment]
Thx for quick answer.

It doesn't matter whether you need a building drone or whether you open the menu with the push of a button, but binding the save menu to a block is a step back in relation to the building system.

**Comment by tsunamayo** [comment]
@Battlepixel how you do that in others game (for survival and creative) ? Surely not with a drone ;)
I was thinking of adding a button to the tab menu.

**Comment by Battlepixel** [comment]
In some like Avorion or Starmade ore FtD you can switch in a build-menü while sitting in the ship ore core. But I hate having a shipcore because its mostly ugly, unnecessary and in worst case you can not replace it,
The building style and save-options we had with the building-drone-Menü were the best solution I have ever seen. But if you dont like the drone, no prob your idea with the button in the tab menü will also do, if you can easy select the entity you want to save or repair.

**Comment by Xerma** [comment]
What if the option were in the ship and hovercraft cores respectively? Or simply make the build core similar in size/placement as the ship core, I would think one of those might work best for all situations

**Comment by Battlepixel** [comment]
I dont think so. 
This means that you always have to install the core in an accessible manner. But the core should be safe from fire, so it should be in the cockpit. Now your cockpit is small and you can only get to the core when the cockpit is open. If you save and load the fighter now, the cockpit is always open, and you are forced to place the core somewhere in the cockpit. For what that all the building drone menu was perfect in terms of a Maxiamler degree of freedom

**Comment by ProPeach** [comment]
Moving the entire drone functionality to a menu in the codex is the best option in my opinion. The functionality of the drone was good, the only issue with it was trying to find it again after you be been building for a while the  respecting a build etc. Moving it's tools to a codex menu eliminates that problem as you can always open the codex wherever you are

**Comment by schlid** [comment]
I agree with Peach, however moving it a hotkey, say f5 for consistency and then projecting the menu from your build gun? It'd be the most streamlined solution and prevent a new codex tab being added unnecessarily.

**Comment by ExodistSKY1** [comment]
THANK YOU!!!!

![20200422134635_1](https://user-images.githubusercontent.com/45324484/80021737-981ea380-84a0-11ea-8b87-0c4f5fd80378.jpg)
This seat is exactly what I was wanting!

Three things I have noticed so far:

![20200422102432_1](https://user-images.githubusercontent.com/45324484/80021855-c9976f00-84a0-11ea-9124-ab718896b981.jpg)
Thin canopy windows and half slab blocks are in the hot bar with an image of the starter block

If you can make this canopy window in brick form that would be fantastic!

![20200422125336_1](https://user-images.githubusercontent.com/45324484/80022001-05cacf80-84a1-11ea-976e-e50b14dcd676.jpg)
The Panel beveled brick cannot be placed the way it used to to create a bordered edge 

![20200422104517_1](https://user-images.githubusercontent.com/45324484/80022090-2a26ac00-84a1-11ea-9157-c6e27fb7c513.jpg)
Possibly pushing the character models back in the seats a little 

Really great update, looking forward to more!
